source/Publish Profiles/Pre-production/phe-dev-reporting.publish.xml

```diff
@@ -91,7 +91,7 @@
       <Value>NTBS_Migration</Value>
     </SqlCmdVariable>
     <SqlCmdVariable Include="NTBS">
-      <Value>NTBS_Live</Value>
+      <Value>NTBS_Dev</Value>
```

Comment:
How odd - this looks like it's been this way for a while

Reply:
Oh, so I think this has been wrong ever since the delete profiles were created, but has only come to the surface now that we've got rid of the non-delete profiles.